packages/devtools_app/lib/src/shared/framework/screen.dart

```diff
@@ -339,6 +348,10 @@ abstract class Screen {
   /// is enabled.
   final bool requiresAdvancedDeveloperMode;
 
+  /// Whether this screen should only be included when the app is a web app with full debugging
```

Comment:
[nit] remove "only" ("Whether this screen should be included when...")

Reply:
Done!